config/routes.rb

```diff
@@ -10,7 +10,9 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
-    resources :companies, only: [:show, :destroy, :update]
+    resources :companies, only: [:show, :destroy, :update] do
+      resources :notes, only: [:index, :create, :destroy]
+    end
     resources :pdfs, only: [:create, :new]
     resource :tools_resources, only: [:edit]
   end
```

Comment:
I am confused about this as well, but maybe because I was just expecting to see one notes section on each company show page. I think this could make sense if we want to maintain multiple notes for each company.